Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -1488,7 +1483,49 @@ TEST_F(SerializerTest, EncodesListenRequestLabels) {
   }
 }
 
-TEST_F(SerializerTest, DecodesListenResponseWithTargetChange) {
+TEST_F(SerializerTest, DecodesMutationResult) {
+  std::vector<FieldValue> transformations({FieldValue::FromBoolean(true),
+                                           FieldValue::FromInteger(1234),
+                                           FieldValue::FromString("string")});
+  MutationResult model(Version(123456789), std::move(transformations));
```

Comment:
This version is repeated several times throughout the test, can you create a local variable for it?

Reply:
Done.